### DISCUSSION TITLE
Is There A Way To Listen Laravel Mail Events On Livewire?

### DISCUSSION THREAD

**Discussion by reasecret** [livewire/livewire#6959] [1. Help]
I've a component for sending batch email to users. I want to listen Laravel's MessageSending and MessageSent events and show email status on this component. Is there any way to do this? (Except broadcasting). I couldn't find anything about this.

**Comment by edalzell** [comment]
> Except broadcasting

Why don't you want to use this? It's the right way to do it, other than polling.

**Reply by reasecret** [reply]
That's right. Just wondered is there another way. Like `Event::listen(MessageSending)` on Livewire component.

**Reply by younus93** [reply]
Were you able to implement this?
